packages/block-editor/src/components/off-canvas-editor/style.scss

```diff
@@ -1,397 +1,17 @@
-		width: 100%;
-	}
-
-	.block-editor-list-view-block-select-button__anchor {
-		position: absolute;
-		right: 0;
-		transform: translateY(-50%);
-		background: rgba($black, 0.1);
-		border-radius: $radius-block-ui;
-		padding: 2px 6px;
-		max-width: 100%;
-		box-sizing: border-box;
-	}
-
-	&.is-selected .block-editor-list-view-block-select-button__anchor {
-		background: rgba($black, 0.3);
-	}
-
-	.block-editor-list-view-block-select-button__lock {
-		line-height: 0;
-		width: 24px;
-		min-width: 24px;
-		margin-left: auto;
+.offcanvas-editor__list-appender__toggle {
+	flex-direction: row;
+	justify-content: flex-start;
+	border: 0;
+	box-shadow: none;
+	margin-left: 24px;
+	margin-top: $grid-unit-10; // List View block SVG icon size.
+
+	&.components-button.components-button {
 		padding: 0;
-		vertical-align: middle;
-	}
-}
-
-.block-editor-list-view-block-select-button__description,
-.block-editor-list-view-appender__description {
-	display: none;
-}
-
-.block-editor-list-view-block__contents-cell,
-.block-editor-list-view-appender__cell {
-	.block-editor-list-view-block__contents-container,
-	.block-editor-list-view-appender__container {
-		display: flex;
 	}
-}
 
-// Chevron container metrics.
-.block-editor-list-view__expander {
-	height: $icon-size;
-	margin-left: $grid-unit-05;
-	width: $icon-size;
-}
-
-// First level of indentation is aria-level 2, max indent is 8.
-// Indent is a full icon size, plus 4px which optically aligns child icons to the text label above.
-$block-navigation-max-indent: 8;
-.block-editor-list-view-leaf[aria-level] .block-editor-list-view__expander {
-	margin-left: ( $icon-size ) * $block-navigation-max-indent + 4 * ( $block-navigation-max-indent - 1 );
-}
-
-.block-editor-list-view-leaf:not([aria-level="1"]) {
-	.block-editor-list-view__expander {
-		margin-right: 4px;
-	}
-}
-
-@for $i from 0 to $block-navigation-max-indent {
-	.block-editor-list-view-leaf[aria-level="#{ $i + 1 }"] .block-editor-list-view__expander {
-		@if $i - 1 >= 0 {
-			margin-left: ( $icon-size * $i ) + 4 * ($i - 1);
-		}
-		@else {
-			margin-left: ( $icon-size * $i );
-		}
-	}
-}
-
-.block-editor-list-view-leaf .block-editor-list-view__expander {
-	visibility: hidden;
-}
-
-// Point downwards when open.
-.block-editor-list-view-leaf[aria-expanded="true"] .block-editor-list-view__expander svg {
-	visibility: visible;
-	transition: transform 0.2s ease;
-	transform: rotate(90deg);
-	@include reduce-motion("transition");
-}
-
-// Point rightwards when closed
-.block-editor-list-view-leaf[aria-expanded="false"] .block-editor-list-view__expander svg {
-	visibility: visible;
-	transform: rotate(0deg);
-	transition: transform 0.2s ease;
-	@include reduce-motion("transition");
-}
-
-.block-editor-list-view-drop-indicator {
-	pointer-events: none;
-
-	.block-editor-list-view-drop-indicator__line {
-		background: var(--wp-admin-theme-color);
-		height: $border-width;
+	&:hover,
+	&:focus {
+		box-shadow: none;
 	}
 }
```

Comment:
Shall we just include an empty file to start with as these changes don't make sense in the context of this PR 🙏